tests/components/alarm_control_panel/conftest.py

```diff
@@ -160,7 +160,7 @@ async def async_setup_entry_init(
         )
         return True
 
-    MockPlatform(hass, f"{TEST_DOMAIN}.config_flow")
+    MockPlatform()
```

Comment:
Not 100% sure what this one was meant to do - but the current implementation is wrong.
Maybe it was meant to be `mock_platform`?

Reply:
Ended up removing the line altogether as didn't seem to server any purpose.